### PR TITLE
chore: Project Addition by blindaks

### PR DESCRIPTION
**Pull request by blindaks**
## Related Issue
#2207 

## Description
Added a simple quiz website based on front-end web development.

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -3499,5 +3499,15 @@
...
+{
+	"github_url": "https://github.com/blindaks/Quiz-App",
+	"title": "Quiz App",
+	"description": "A quiz app based on front wnd development questions.",
+	"tech": [
+	        "html",
+	        "css",
+	        "javascript"
+	]
+}
```
Comment (priyankarpal) [inline DB/projects.json:3512]: not here. check the docs https://projectshut.vercel.app/docs/ & sample code below

```
{
  "github_username": "YOUR_GITHUB_USERNAME",  
  "Projects": [
    {
      "github_url": "1st_PROJECT_LINK",
      "title": "PROJECT_NAME",
      "description": "PROJECT_DESCRIPTION",
      "tech": ["tech1", "tech2"]
    },
    {
      "github_url": "2nd_PROJECT_LINK",
      "title": "PROJECT_NAME",
      "description": "PROJECT_DESCRIPTION",
      "tech": ["tech1", "tech2"]
    }
  ]
}
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json (hunk 1 above)
Comment (blindaks) [reply DB/projects.json:3512]: Do you want me to add "Projects:" line? So that I can put any future projects there?

Comment (priyankarpal) [reply DB/projects.json:3512]: yes, in future you need to add ``` {
      "github_url": "2nd_PROJECT_LINK",
      "title": "PROJECT_NAME",
      "description": "PROJECT_DESCRIPTION",
      "tech": ["tech1", "tech2"]
    }```
inside the projects array

Comment (blindaks) [reply DB/projects.json:3512]: Okay

File: DB/projects.json [hunk 2]
```diff
@@ -3499,5 +3499,16 @@
...
-  }
+  },
+{
+  "github_username": "blindaks",  
+  "Projects": [
+    {
+      "github_url": "https://github.com/blindaks/Quiz-App",
+      "title": "Quiz App",
+      "description": "A quiz app based on front wnd development questions.",
+      "tech": ["HTML", "CSS", "JavaScript"]
```
Comment (priyankarpal) [inline DB/projects.json:3510]: convert to small case 
```suggestion
      "tech": ["html", "css", "javascript"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]

**Pull request merged**